srcpkgs/pass-import/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'pass-import'
 pkgname=pass-import
-version=3.2
-revision=2
+version=3.3
+revision=1
 build_style=python3-module
 # This test requires gnome-keyring which requires running dbus
 make_check_args="--deselect=tests/imports/test_parse.py"
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools python3-pypandoc python3-requests python3-yaml
+ python3-zxcvbn"
```

Comment:
Please switch `distfiles` to `PYPI_SITE` https://pypi.org/project/pass-import/#files, `pypandoc` is not very welcome in hostmakedepends

Reply:
https://github.com/roddhjav/pass-import/issues/192

Reply:
Yes will do this evening

Reply:
Changed to PYPI_SITE and removed the makedoc from setup.py. There are no tests in these sources, so do I drop the checkdepends altogether?